plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -516,6 +525,12 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
                           ("txid", trx->id())
                           ("why",std::get<fc::exception_ptr>(response)->what()));
                }
+
+               if (_trx_trace_dump_log.is_enabled(fc::log_level::debug)) {
+                   //Dump transaction trace
+                   auto entire_trace = std::get<fc::exception_ptr>(response);
+                   fc_dlog(_trx_trace_dump_log, "[TRX_TRACE] ${entire_trace} ", ("entire_trace", entire_trace));
```

Comment:
Should add "Speculative execution is REJECTING tx:"

Reply:
Ok. I am reverting to previous version of changes, i.e., `if` and `else` have different log messages. 

The log message in `if` block would be `"[TRX_TRACE] Block ${block_num} for producer ${prod} is REJECTING tx: ${entire_trace}` , and the log message in `else` block would be `[TRX_TRACE] Speculative execution is REJECTING tx: ${entire_trace}`. Does it look ok?

Reply:
Fixed.